src/ring/middleware/session_timeout.clj

```diff
@@ -4,6 +4,19 @@
 (defn- current-time []
   (quot (System/currentTimeMillis) 1000))
 
+(defn idle-session-timeout
```

Comment:
Public functions should have a docstring. The naming scheme used in Ring for response modifying functions is `<something>-response`, so in this case it would be `idle-session-timeout-response`.

Reply:
I fixed the name but my first instinct is to mark it private instead of adding a docstring. I think it will have much the same docstring as the public function.